src/api/octane.ts

```diff
@@ -339,12 +339,22 @@ export const getTotalAccruedRevenue = makeApiGETEndpoint<
   unknown // Undefined failure type
 >(getTotalAccruedRevenueUrl);
 
-export const getDailyAccruedRevenueUrl: UrlFactory = (base = API_BASE) =>
-  `${base}/ecp/daily_accrued_revenue`;
+export const getDailySpendUrl: UrlFactory = (base = API_BASE) =>
```

Comment:
I know the default is daily, but we can use this to fetch usage by hour too.  

It seems I didn't update the usage_by_time endpoint fully.  I will do that now.

After I make the change, we'll need to regen the types here. Can you also update the names here too?